src/App.tsx

```diff
@@ -9,42 +9,41 @@ import RecipeList from "./containers/RecipeList";
 import RecipeDetail from "./containers/RecipeDetail";
 import AddRecipeForm from "./components/AddRecipeForm";
 
-class App extends Component {
-  constructor(props) {
-    super(props);
+interface IAppState {
+  isAddingRecipe: boolean
+  selectedRecipe: boolean
+}
 
-    this.state = {
-      isAddingRecipe: false
-    };
+class App extends Component<{}, IAppState> {
+  state: IAppState = {
+    isAddingRecipe: false,
+    selectedRecipe: false
   }
 
   handleAddRecipe = () => {
     this.setState({ isAddingRecipe: true, selectedRecipe: false });
   };
 
-  handleAddRecipeInput = e => {
-    let { name, value } = e.target;
-    if (name === "price") value = +value;
-    this.setState({ [name]: value });
-  };
+  // handleAddRecipeInput = e => {
+  //   let { name, value } = e.target;
+  //   if (name === "price") value = +value;
+  //   this.setState({ [name]: value });
+  // };
```

Comment:
Neither of these were actively being used so I just commented them out for now to get things to compile!